lib/web_ui/lib/src/engine/compositor/canvas.dart

```diff
@@ -14,7 +14,7 @@ class SkCanvas {
   int get saveCount => skCanvas.callMethod('getSaveCount');
 
   void clear(ui.Color color) {
-    skCanvas.callMethod('clear', <int>[color.value]);
+    skCanvas.callMethod('clear', <js.JsObject>[makeSkColor(color)]);
```

Comment:
Driveby comment here from a Dart noob: Is there not a dart type for float32arrays? http://commondatastorage.googleapis.com/dartlang-api-docs/10315/dart_html/Float32Array.html

Reply:
There is. Does `CanvasKit.Color` simply construct a `Float32Array`?

Reply:
Looks like it is: https://github.com/google/skia/blob/41fe8bc3500ea15f1d14362ca4aeef21aecb2314/modules/canvaskit/helper.js#L26

We can do this faster in plain Dart code. However, I'm still uneasy about having to convert Flutter colors, which are plain `int` values, to `Float32Array`s :/

Reply:
I changed this to create Float32Arrays.